xray/src/generation.rs

```diff
@@ -227,14 +257,22 @@ impl ColoringStrategy for IntensityColoringStrategy {
                 {
```

Comment:
How about instead of `match self.per_column_data.entry(...)` we do `self.per_column_data.entry(...).or_default()`? We seem to be doing very similar stuff in both branches of the `match`.

Reply:
Good idea, I changed it.